unic/ucd/name/tests/basic_tests.rs

```diff
@@ -62,7 +59,6 @@ fn test_name_cmp() {
 
     // == NR1 ==
 
-    // FIXME: use decomposed jamo short names instead
```

Comment:
Should we add more tests here for sort ordering of JAMO vs JAMO and JAMO vs others?

Reply:
Sure, will add a few more tests!